modules/ROOT/pages/customize-layouts.adoc

```diff
@@ -70,13 +71,13 @@ Once your changes are made, save the new layout using a new name and then export
 . Click on *Save*.
 . Click on the *Export* button.
 
-== Import the _myCustomLayout_ into the portal
+== Import the _myCustomLayout_ into the Bonita Administrator Application
 
-. Open Bonita Portal, and again log in with username and password of someone mapped to the Administrator profile.
+. Log in with someone mapped to the Administrator profile.
```

Comment:
Log in with someone mapped to the Administrator profile. => Log in with a users having the Administrator profile.

its not a date to go with someone 👯

Reply:
Modified this one as well